exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/QaServiceImpl.java

```diff
@@ -108,6 +121,18 @@ public void createPublicApiHandlers(Node node, Router router) {
     controller.mountApi(router);
   }
 
+  /**
+   * Increments the afterCommit counter so the number of times this method was invoked is stored
+   * in it.
+   */
+  @Override
+  public void afterCommit(BlockCommittedEvent event) {
+    long seed = RANDOM.nextLong();
```

Comment:
Shan't we just use the blockchain height from the event — it is guaranteed to be unique? It will be simpler in terms of code and the reader won't have to think about the probability of a certain value appearing again in the sequence after _N_ invocations. It must be small given `N << 2^64` though